### PR TITLE
Created a json file for the anagram exercises

### PR DESCRIPTION
**Pull request by robyngolding**
exercism/todo#68 pull request part of @codebar @ladieswhocode for the 24 pull requests event.

### CODE REVIEW

**Comment by suzannehamilton** [comment]
Oops, syntax error! (And a duplicated test.)

We're fixing it. :)

**Comment by kytrinyx** [comment]
Cool :) Let me know if you need a hand at any point!

**Comment by robyngolding** [comment]
Hi I'm not sure, but I think this is okay to be merged now? Let me know if there's something left I need to do. Thank you :)

File: anagram.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        "these string argument cases if passing individual arguments is",
+        "idiomatic in that language.",
+        "Some tests have a \"comment\" property, which provides more",
+        "information that could be added to the test case as a code comment."
+    ],
+    "cases": [
+        {
+            "description": "no matches",
+            "subject": "diaper",
+            "possible matches": [ "hello", "world", "zombies", "pants"],
```
Comment (kytrinyx) [inline anagram.json]: I'd like to avoid having spaces in JSON keys. How about calling this `"input"`?

**Comment by kytrinyx** [comment]
Nice! I have one tiny comment, but this is looking great.

Once you're done, would you mind squashing the commits? http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Comment by kytrinyx** [comment]
I was thinking about the name of the `input` key... how about `candidates`?

This still has 8 commits. Would you mind squashing them please? (I can squash them for you if you like, just let me know).

**Comment by suzannehamilton** [comment]
@robyngolding - do you want to have a go at making the change and squashing the commits? Give me a shout if you want a hand, and we could email or pair over Skype.

**Comment by robyngolding** [comment]
Hi Suzy I'll try it tonight. I'll let you know if I get stuck, thanks so much!

**Comment by robyngolding** [comment]
I think this is ready now, candidates instead of input and squashed commits.

**Comment by kytrinyx** [comment]
This is great, thank you!

**Pull request merged**